rdt/transformers/numerical.py

```diff
@@ -40,7 +40,8 @@ class FloatFormatter(BaseTransformer):
         missing_value_replacement (object):
             Indicate what to replace the null values with. If an integer or float is given,
             replace them with the given value. If the strings ``'mean'`` or ``'mode'``
-            are given, replace them with the corresponding aggregation. Defaults to ``mean``.
+            are given, replace them with the corresponding aggregation and if ``'random'``
+            replace each null value with a random value from the data. Defaults to ``random``.
```

Comment:
can be confusing `replace each null value with a random value from the data.` I feel. Maybe something like `replace each null value with a random value in the data range.`